detection_rules/etc/version.lock.json

```diff
@@ -4116,25 +4130,25 @@
       }
     },
     "rule_name": "Remote SSH Login Enabled via systemsetup Command",
-    "sha256": "7331f31c20c18949ae8238427d264c121096770ea65404d5eb21382668185a67",
+    "sha256": "b155970c7e73cb523e459fa39f92799a94642e17e5c4325815e6d66942409c8e",
     "type": "query",
-    "version": 100
+    "version": 101
   },
   "5aee924b-6ceb-4633-980e-1bde8cdb40c5": {
     "min_stack_version": "8.3",
     "previous": {
       "7.16": {
         "max_allowable_version": 99,
         "rule_name": "Potential Secure File Deletion via SDelete Utility",
-        "sha256": "d50e3a802eeb576df60ee993ea3942d6b4ce183cab3c5b63982081fbc72b997d",
+        "sha256": "b42e9f2369de3fa9727f635d630089197d955d8b0e0a1dcb89bcd880066ea6ab",
```

Comment:
Forked rule changed when all rules are min-stacked at 8.3 and thus changes would not backport to this branch.